todo/src/components/Todo.tsx

```diff
@@ -0,0 +1,14 @@
+
+const Todo = ({task, completed, handleCheck, idx, removeTask}: {task: string, completed: boolean, handleCheck: (id: number) => void, idx: number, removeTask: (id: number) => void}) => {
```

Comment:
Instead of directly specifying types here, let's use interfaces and assign values to props.

Reply:
changes done